src/scripts/main.js

```diff
@@ -354,7 +354,33 @@ const people = [
   },
 ];
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+function calculateAge(born, died) {
+  return Math.ceil(died - born);
```

Comment:
Using `Math.ceil` for calculating age is unnecessary and may lead to incorrect results. Simply subtracting 'born' from 'died' will give the correct age.